api/controllers/AuthController.js

```diff
@@ -68,12 +68,8 @@ module.exports = {
    * @param {Object} res
    */
   logout: function (req, res) {
-    req.logout();
-
-    // mark the user as logged out for auth purposes
-    req.session.authenticated = false;
-
-    res.redirect('/');
+    req.session.destroy();
```

Comment:
Tell you what, can you do a check here if we are websockets, and if we aren't, do a redirect. Just would make things easier if we do a Javascript-free frontend in the future.